### DISCUSSION TITLE
Miningcore/Alphaxcore - shares table - column source empty

### DISCUSSION THREAD

**Discussion by tdvoracek** [coinfoundry/miningcore#1009] [Q&A]
Hello, 
is a problem when source column is empty? Same on alphaxcore and miningcore.
thanks
![shares1](https://user-images.githubusercontent.com/47913735/143320116-7e4383bc-f7bd-4ea6-894d-0063306983ec.jpeg)

**Comment by belmix** [comment]
"clusterName": "eu1", in pool_config.json

**Reply by tdvoracek** [reply]
ok, now i can see eu1 in db, what does that mean please? 
is that important or just name and system can work without?
is there also cluster for USA? 

thanks
